src/bootstrap/check.rs

```diff
@@ -268,51 +293,64 @@ pub fn krate(build: &Build,
              compiler: &Compiler,
```

Comment:
this function has a doc comment that mentions "Cargo.lock"; that part should be changed to say `cargo metadata`